researchstack-mpower/src/main/res/values/themes.xml

```diff
@@ -7,14 +7,14 @@
         <item name="alertDialogTheme">@style/Theme.MP.Dialog</item>
         <item name="android:alertDialogTheme">@style/Theme.MP.Dialog</item>
 
-        <item name="android:textColor">@color/appTextDark</item>
-        <item name="android:textColorPrimary">@color/appTextDark</item>
+        <item name="android:textColor">@color/rsb_warm_gray</item>
```

Comment:
Did you test surveys after making all these changes?

Reply:
Briefly. What I did was move them from /app/ to here since they must have been missed in a previous refactor.

I searched for all the themes to make sure they are now declared in only one place, and diffed the old theme file in app and this one 

![image](https://user-images.githubusercontent.com/7097368/49254492-33136b00-f3de-11e8-9513-bae3d63149f6.png)